Network/Public/Include/Network/Message.hpp

```diff
@@ -27,7 +27,9 @@ struct Message
         MessageHistoryRequest,
         MessageStoreRequest,
         RegistrationRequest,
-        RegistrationAnswer
+        RegistrationAnswer,
+        UserMessageDelete,
```

Comment:
It is better to follow the similarity: UserMessageDeleteRequest and UserMessageDeleteAnswer. When the client send the message to the server - it's request; if server send message - it is answer for client.